.eslintrc.json

```diff
@@ -1,6 +1,5 @@
 {
   "extends": "./lib/index.js",
-  "parser": "@typescript-eslint/parser",
```

Comment:
https://github.com/eslint/eslint/pull/11388/files#diff-33bbe98f532eb757e2789f1a3549c9caR41

So, our `./lib/index.js` specifies the `parser`.